docs/markdown/Continuous-Integration.md

```diff
@@ -44,6 +44,65 @@ script:
   - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then SDKROOT=$(xcodebuild -version -sdk macosx Path) meson builddir && ninja -C builddir test; fi
 ```
 
+## CircleCi for Linux (with Docker)
+
+For CI on Linux, [CircleCi](https://circleci.com/) is probably
+your best bet. Here's a sample `yml` file for use with that.
+
+```yaml
+version: 2.1
+
+executors:
+  # Your dependencies would go in the docker images that represent
+  # the Linux distributions you are supporting
+  meson_ubuntu_builder:
+    docker:
+      - image: michaelbadcruble/ubuntu-sys
```

Comment:
maybe the image should be generic so readers realize they need to choose one like
```yml
    docker:
      - image: your_dockerhub_username/ubuntu-sys
```

Reply:
Sure thing.  Update applied.

Reply:
Ok @scivision does that mean I should revert the Travis example for you to work on?

Reply:
Ok reverted the Travis example so you can work on it.

Reply:
no need to revert, I also want to add Github Actions so go ahead and I'll do it later.